CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Change Log
 
 ## [Unreleased]
+### Fixed
```

Comment:
I changed my mind (Sorry for the back and forth) but maybe we should define this change as a feature for a more positive vision.
```suggestion
### Added
```